content/influxdb/v1.8/query_language/explore-data.md

```diff
@@ -713,7 +713,7 @@ SELECT_clause FROM_clause [WHERE_clause] GROUP BY [* | <tag_key>[,<tag_key]]
 The order of the [tag keys](/influxdb/v1.8/concepts/glossary/#tag-key) is irrelevant.
 
 If the query includes a [`WHERE` clause](#the-where-clause) the `GROUP BY`
-clause must appear after the `WHERE` clause.
+clause must appear after the `WHERE` clause. 
```

Comment:
@ayang64 @jsteenb2
per the workaround in [EAR 1387](https://github.com/influxdata/EAR/issues/1387)
it appears we can include the WHERE clause after the GROUP BY clause.

Question:
Do we remove this note altogether, or does this note only not apply with subqueries --and is still valid for "main" queries?